site/src/pages/WorkspacePage/AppStatuses.tsx

```diff
@@ -304,7 +304,10 @@ export const AppStatuses: FC<AppStatusesProps> = ({
 													},
 												}}
 											>
-												<OpenInNew sx={{ mr: 0.5 }} />
+												<ExternalLinkIcon
+													className="size-icon-xs"
+													style={{ marginRight: "4px" }}
```

Comment:
can you turn this into a tailwind class pls?

Reply:
Sure! But let me do that in a diff PR just to convert emotion styles into tailwindcss related ones ;)